Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6832,6 +6846,20 @@ UpgradeKnownHostsToHTTPSEnabled:
     WebKit:
       default: true
 
+UpgradeMixedContentEnabled:
+  type: bool
+  status: unstable
+  category: networking
+  humanReadableName: "Upgrade upgradable mixed content"
+  humanReadableDescription: "Upgrade upgradable mixed content"
+  defaultValue:
+    WebKitLegacy:
+      default: false
+    WebKit:
+      default: false
+    WebCore:
+      default: false
```

Comment:
Do you plan to enable these eventually? Just being conservative for starters?

Reply:
Yeah, I have a follow-up patch ready that'll enable this, but directly enabling it was a bit noisy, so I decided to separate them.